samples/aspnetcore/SwaggerSample/Properties/launchSettings.json

```diff
@@ -19,7 +19,7 @@
     "BasicSample": {
```

Comment:
For completeness, let's also change this to "SwaggerSample". I must have simply done a _Copy & Paste_ from the basic example. @_@